.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-and-build:
+    runs-on: ubuntu-latest
+    env:
+      LIFECYCLE_VERSION: 0.7.0
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up go
+        uses: actions/setup-go@v2-beta
+        with:
+          go-version: '1.13'
+      - name: Set up go env
+        run: |
+          echo "::set-env name=GOPATH::$(go env GOPATH)"
+          echo "::add-path::$(go env GOPATH)/bin"
+        shell: bash
+      - name: Install jq
+        run: |
+          mkdir -p deps/bin
+          curl -s -L -o deps/bin/jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64
+          chmod +x deps/bin/jq
+          echo "::add-path::${PWD}/deps/bin"
+      - name: Test
+        run: make test
+      - name: Build
```

Comment:
We should also `build-windows` here to ensure it continues to compile even though we don't need to upload the not-yet-usable artifact.